src/octoprint/printer/standard.py

```diff
@@ -718,6 +722,17 @@ def get_file_position(self):
 
         return self._comm.getFilePosition()
 
+    def addMarking(self, type):
```

Comment:
Sorry for not catching that earlier, but this should actually be `add_marking`. Yes, a lot of OctoPrint's older methods, e.g. the one referenced just above this, still make use of camelCase, but new stuff should stick to Python's established standard of snake_case (and I'm also trying to make old stuff fit whenever I do refactoring in the general region as well).